packages/create-discord-bot/template/TypeScript/package.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "[REPLACE-NAME]",
+	"version": "0.1.0",
+	"private": true,
+	"type": "module",
+	"scripts": {
+		"start": "node --require dotenv/config dist/index.js",
+		"test": "tsc"
+	},
+	"dependencies": {
+		"discord.js": "^14.11.0",
+		"dotenv": "^16.0.3"
+	},
+	"devDependencies": {
+		"@sapphire/ts-config": "^4.0.0",
+		"typescript": "^5.0.4"
```

Comment:
Also needs `@types/node`

Reply:
We should probably also include or ask for ESLint.

Reply:
Including eslint-config-neon would be a good goal, but that can be in a later iteration since this is already big.